server/jetstream_cluster_test.go

```diff
@@ -6456,6 +6456,10 @@ func (c *cluster) waitOnPeerCount(n int) {
 	c.t.Helper()
 	c.waitOnLeader()
 	leader := c.leader()
+	for leader == nil {
+		c.waitOnLeader()
+		leader = c.leader()
```

Comment:
Just pushes the problem, same thing can happen after this line.

Reply:
That was my first though but look, it is a "for" loop. And as he explains in description, waitOnLeader() will Fatal() the test if there is no leader.

Reply:
A gotcha, I skipped over that part. Apologies.